code/__DEFINES/mob_defines.dm

```diff
@@ -240,6 +240,7 @@
 #define isbot(A)			(istype((A), /mob/living/simple_animal/bot))
 #define isguardian(A)		(istype((A), /mob/living/simple_animal/hostile/guardian))
 #define isnymph(A)      	(istype((A), /mob/living/simple_animal/diona))
+#define iscaterpillar(A)			(istype((A), /mob/living/simple_animal/nian_caterpillar))
```

Comment:
```suggestion
#define iscaterpillar(A)	(istype((A), /mob/living/simple_animal/nian_caterpillar))
```
web editing tends to ruin these, im trying to pull the `(istype((A),` inline with the others.